modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The {lcao} will do the following:
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation tool, which does the following:
+** Stops the cluster operators
```

Comment:
what does it mean stops clusters operators? IIRC it stops the kubernetes agent (kubelet) and the container runtime (crio) so, then all Pods running on OpenShift are stopped too. I mean, creates a clean environment to create the image. cc/ @leo8a